
=== Test project_root: /tmp/pytest-of-runner/pytest-0/project60


Invoking dbt with ['run']
[0m14:25:06  Running with dbt=1.10.5
[0m14:25:06  Registered adapter: firebolt=1.9.4
[0m14:25:06  Unable to do partial parsing because saved manifest not found. Starting full parse.
[0m14:25:06  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- seeds.test
[0m14:25:06  Found 2 models, 464 macros, 1 unit test
[0m14:25:06  
[0m14:25:06  Concurrency: 1 threads (target='default')
[0m14:25:06  
[0m14:25:06  1 of 2 START sql view model my_upstream_model .................................. [RUN]
[0m14:25:06  1 of 2 OK created sql view model my_upstream_model ............................. [[32mSUCCESS[0m in 0.09s]
[0m14:25:06  2 of 2 START sql view model my_model ........................................... [RUN]
[0m14:25:06  2 of 2 OK created sql view model my_model ...................................... [[32mSUCCESS[0m in 0.09s]
[0m14:25:06  
[0m14:25:06  Finished running 2 view models in 0 hours 0 minutes and 0.30 seconds (0.30s).
[0m14:25:06  
[0m14:25:06  [32mCompleted successfully[0m
[0m14:25:06  
[0m14:25:06  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 NO-OP=0 TOTAL=2


Invoking dbt with ['test']
[0m14:25:06  Running with dbt=1.10.5
[0m14:25:06  Registered adapter: firebolt=1.9.4
[0m14:25:06  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- seeds.test
[0m14:25:06  Found 2 models, 464 macros, 1 unit test
[0m14:25:06  
[0m14:25:06  Concurrency: 1 threads (target='default')
[0m14:25:06  
[0m14:25:06  1 of 1 START unit_test my_model::test_my_model ................................. [RUN]
[0m14:25:07  1 of 1 ERROR my_model::test_my_model ........................................... [[31mERROR[0m in 0.32s]
[0m14:25:07  
[0m14:25:07  Finished running 1 unit test in 0 hours 0 minutes and 0.40 seconds (0.40s).
[0m14:25:07  
[0m14:25:07  [31mCompleted with 1 error, 0 partial successes, and 0 warnings:[0m
[0m14:25:07  
[0m14:25:07  [31mFailure in unit_test test_my_model (models/unit_tests.yml)[0m
[0m14:25:07    Runtime Error in unit_test test_my_model (models/unit_tests.yml)
  An error occurred during execution of unit test 'test_my_model'. There may be an error in the unit test definition: check the data types.
   Runtime Error
    Error executing query:
    {
      "errors": [
        {
          "description": "CREATE DIMENSION TABLE AS SELECT is not supported in Firebolt Core"
        }
      ],
      "query": {
        "query_id": "f07e9b24-c2fc-460a-b1b8-a626b9c3fe30",
        "query_label": null,
        "request_id": "379050c9-10d4-4aa8-8b8e-77a52dbd967c"
      },
      "statistics": {
        "elapsed": 0.0
      }
    }
    
[0m14:25:07  
[0m14:25:07  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 NO-OP=0 TOTAL=1
